java/opendcs/src/main/java/decodes/cwms/CwmsTimeSeriesDAO.java

```diff
@@ -788,6 +792,16 @@ public void saveTimeSeries(CTimeSeries ts) throws DbIoException,
                     qualities, num2write, CwmsConstants.REPLACE_MISSING_VALUES_ONLY,
                     overrideProtection, versionDate, false);
             }
+
+            TimeSeriesIdentifier tsIdCached = cache.getByUniqueName(tsId.getUniqueString());
+            if (tsIdCached == null)
+            {
+                // If this key is not in the cache it was freshly saved. Add it to the
+                // Cache now for anything that may use it.
```

Comment:
Won't the caching logic elsewhere cache the tsId on the next request?

Reply:
It appeared that it wasn't; however, this is leftover code from the attempt. That said since we have it now, it makes sense to add it to the cache and save several additional database calls.

Reply:
I'm a bit more confused as to why this is showing up here since this was derived from the CWMS Flyway... will need to rebase.